src/test/java/ikysil/training/ws/api/v1/OrdersResourceTest.java

```diff
@@ -24,27 +37,90 @@
 @RunWith(SpringRunner.class)
 public class OrdersResourceTest {
 
+    OrderDao dao = new InMemoryOrderDao();
+
     public static final String BASE_URL = "/ws/api/v1/orders/";
 
     @LocalServerPort
     private int port;
 
+    private OrderDto order1 = new OrderDto("1144", "twenty one", "none", "3129", "firstuser@gmail.com");
+    private OrderDto order2 = new OrderDto("3257", "twenty two", "none", "4024", "seconduser@gmail.com");
+    private OrderDto newOrder = new OrderDto("3257","thirty three", "noneone", "2440", "newuser@gmail.com");
+
     @Before
     public void setUp() {
         RestAssured.basePath = BASE_URL;
         RestAssured.port = port;
     }
 
+    @Before
+    public void savingOK() {
+        given().contentType(ContentType.JSON)
+                .body(order1, ObjectMapperType.JACKSON_2)
+                .when()
+                .post()
+                .then()
+                .statusCode(200);
+        given().contentType(ContentType.JSON)
+                .body(order2, ObjectMapperType.JACKSON_2)
+                .when()
+                .post()
+                .then()
+                .statusCode(200);
+    }
+
+    @Test
+    public void getByIdTest() {
+        given().contentType(ContentType.JSON)
```

Comment:
serialize what you receive and assert it equals to what you expect `order1` and `order2`